src-web/tableDefinitions/grcPoliciesViewDef.js

```diff
@@ -98,6 +98,8 @@ export default {
     {
       label: 'automation',
       msgKey: 'table.header.automation',
+      resourceKey: 'policyAutomations.metadata.name',
```

Comment:
Thanks! This should be singular "policyAutomation"

Reply:
Also, since it's an object you'll need to add:
```yaml
sortLabel: 'automations.rawData'
```
To let it know where to look inside the row data.

Reply:
I tested it locally, and after those two changes the sorting seems to work well.